packages/spatial/src/renderer/components/MeshComponent.ts

```diff
@@ -40,7 +40,7 @@ import { useResource } from '../../resources/resourceHooks'
 import { addObjectToGroup, removeObjectFromGroup } from './GroupComponent'
 
 export const MeshComponent = defineComponent({
-  name: 'Mesh Component',
+  name: 'MeshComponent',
```

Comment:
Are we sure these names aren't being used for the UI in the studio?

Reply:
They aren't. We have i18n that uses the json ID